### PR TITLE
adaptive

### PR DESCRIPTION
**Pull request by Adil2331**
1. adaptive to traffic and task blocks
2. redesigned adaptive menu component

### CODE REVIEW

File: src/components/Analytics/Analytics.tsx [hunk 1]
```diff
@@ -11,18 +11,23 @@ import sun from "../../assets/img/sun.svg";
...
+import { setNav } from "../Redux/menuSlice";
+import OpenMenu from "../Menu/OpenMenu";
 
 const Analytics = () => {
   const dispatch = useDispatch();
+  const [state, setstate] = useState(true);
 
-  const theme = useAppSelector((state) => state.theme);
+  const theme = useAppSelector((state) => state.theme.theme);
   console.log("tehe", theme);
```
Comment (dnaukhan) [inline src/components/Analytics/Analytics.tsx]: remove console.log

File: src/components/Analytics/analytics.modul.scss [hunk 2]
```diff
@@ -48,35 +57,51 @@
   }
 }
 
+// @media (max-width: 660px) {
```
Comment (dnaukhan) [inline src/components/Analytics/analytics.modul.scss]: remove commented code

File: src/components/Task/Task.tsx [hunk 3]
```diff
@@ -41,14 +41,15 @@ const Task = () => {
...
-            }}
+            className="task-inner"
+            // style={{
+            //   display: "flex",
+            //   flexDirection: "row",
+            //   width: "712px",
+            //   height: "56px",
+            //   alignItems: "center",
+            //   justifyContent: "space-between",
+            // }}
```
Comment (dnaukhan) [inline src/components/Task/Task.tsx]: remove commented code

File: src/components/Traffic/Traffic.tsx [hunk 4]
```diff
@@ -11,16 +12,16 @@ const Traffic = () => {
...
+      // style={{
+      //   display: "flex",
+      //   width: "344px",
+      //   height: "360px",
+      //   borderRadius: "16px",
+      //   boxShadow:
+      //     "0px 0px 2px rgba(145, 158, 171, 0.2), 0px 12px 24px -4px rgba(145, 158, 171, 0.12)",
+      //   flexDirection: "column",
+      //   // background: "rgba(255, 255, 255, 1)",
+      // }}
```
Comment (dnaukhan) [inline src/components/Traffic/Traffic.tsx]: remove commented code

**Review by dnaukhan — CHANGES_REQUESTED** [review]

**Review by dnaukhan — APPROVED** [review]

**Review by Kenichi-power — APPROVED** [review]

**Pull request merged**